src/palindrom.js

```diff
@@ -607,18 +563,8 @@ var Palindrom = (function () {
     makeInitialConnection(this);
   }
 
-  Palindrom.prototype = Object.create(EventDispatcher.prototype); //inherit EventTarget API from EventDispatcher
-
   var dispatchErrorEvent = function (palindrom, error) {
```

Comment:
I would say we should now rename this function, as there will be no event dispatched :)